assets/js/modules/sign-in-with-google/components/common/AnyoneCanRegisterDisabledNotice.js

```diff
@@ -72,48 +74,49 @@ export default function AnyoneCanRegisterDisabledNotice( { className } ) {
 	}
 
 	return (
-		<SettingsNotice
-			className={ classnames(
-				'googlesitekit-registration-disabled-notice',
-				'googlesitekit-anyone-can-register-disabled-notice',
-				className
-			) }
-			type={ TYPE_INFO }
-			Icon={ InfoIcon }
-			dismiss={ ANYONE_CAN_REGISTER_DISABLED_NOTICE }
-			dismissLabel={ __( 'Got it', 'google-site-kit' ) }
-			notice={ createInterpolateElement(
-				sprintf(
-					/* translators: %1$s: Setting name, %2$s: Sign in with Google service name */
-					__(
-						'Enable the %1$s setting to allow your visitors to create an account using the %2$s button. <br/>Visit <a>WordPress settings</a> to manage this setting.',
-						'google-site-kit'
-					),
-					isMultisite
-						? __( '“Allow new registrations”', 'google-site-kit' )
-						: __( '“Anyone can register”', 'google-site-kit' ),
-					_x(
-						'Sign in with Google',
-						'Service name',
-						'google-site-kit'
-					)
-				),
-				{
-					a:
-						! canManageOptions && isMultisite ? (
-							<span />
-						) : (
-							<Link key="link" href={ generalSettingsURL } />
+		<div className="googlesitekit-registration-disabled-notice">
+			<Notice
+				type={ Notice.TYPES.INFO }
+				description={ createInterpolateElement(
+					sprintf(
+						/* translators: %1$s: Setting name, %2$s: Sign in with Google service name */
+						__(
+							'Enable the %1$s setting to allow your visitors to create an account using the %2$s button. <br/>Visit <a>WordPress settings</a> to manage this setting.',
+							'google-site-kit'
 						),
-					br:
-						breakpoint === BREAKPOINT_XLARGE ||
-						breakpoint === BREAKPOINT_DESKTOP ? (
-							<br />
-						) : (
-							<Fragment />
-						),
-				}
-			) }
-		/>
+						isMultisite
+							? __(
+									'“Allow new registrations”',
+									'google-site-kit'
+							  )
+							: __( '“Anyone can register”', 'google-site-kit' ),
+						_x(
+							'Sign in with Google',
+							'Service name',
+							'google-site-kit'
+						)
+					),
+					{
+						a:
+							! canManageOptions && isMultisite ? (
+								<span />
+							) : (
+								<Link key="link" href={ generalSettingsURL } />
+							),
+						br:
+							breakpoint === BREAKPOINT_XLARGE ||
+							breakpoint === BREAKPOINT_DESKTOP ? (
+								<br />
+							) : (
+								<Fragment />
+							),
+					}
+				) }
+				dismissButton={ {
+					label: __( 'Got it', 'google-site-kit' ),
+					onClick: dismissNotice,
```

Comment:
In `ProductIDSettings` we use an arrow function instead of a callback: https://github.com/google/site-kit-wp/blob/5f9f0b879fa3f86cfb419ec82764e8505640d735/assets/js/modules/reader-revenue-manager/components/settings/ProductIDSettings.js#L89-L94
We should use the same approach in both places. If we go with `useCallback`, then we need to update that component to use it as well. Or if we don't need that, we should update this one to use arrow function as well.

Reply:
Thanks @eugene-manuilov I have removed the callback and used an arrow function against onClick.